model/src/main/kotlin/Scope.kt

```diff
@@ -32,10 +32,9 @@ data class Scope(
         val name: String,
 
         /**
-         * A flag to indicate whether this scope is delivered along with the product, i.e. distributed to external
-         * parties.
+         * A flag to indicate whether the dependencies in this scope are distributed along with the product.
          */
-        val delivered: Boolean,
+        val distributed: Boolean,
```

Comment:
Consider adding `@JsonAlias("delivered")` for backwards compatibility. Otherwise we cannot parse old analyzer results anymore.

Reply:
Thanks, done.